src/test/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModelTest.java

```diff
@@ -163,8 +164,135 @@ public void testJavadocChild() {
+
+        Assert.assertNotNull("Expected a non-null identifier node here", node);
+        Assert.assertEquals("Expected identifier token",
+            TokenUtils.getTokenId("IDENT"), node.getType());
+
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        final Object treeModel = parseTree.getValueAt(node, 0);
+        final String type = (String) parseTree.getValueAt(node, 1);
+        final int line = (int) parseTree.getValueAt(node, 2);
+        final int column = (int) parseTree.getValueAt(node, 3);
+        final String text = (String) parseTree.getValueAt(node, 4);
+
+        Assert.assertEquals("Node should be an Identifier", "IDENT", type);
+        Assert.assertEquals("Class identifier should start on line 4", 4, line);
+        Assert.assertEquals("Class name should start from column 6", 6, column);
+        Assert.assertEquals("Wrong class name", "InputJavadocAttributesAndMethods", text);
+        Assert.assertNull("Root node should have null value", treeModel);
+
+        try {
+            parseTree.getValueAt(node, parseTree.getColumnCount());
+            Assert.fail("IllegalStateException expected");
+        }
+        catch (IllegalStateException ex) {
+            Assert.assertEquals("Unknown column", ex.getMessage());
+        }
+
+    }
+
+    @Test
+    public void testGetValueAtDetailNode() {
+        final DetailAST commentContentNode = tree.getFirstChild().getNextSibling().getFirstChild();
+        Assert.assertNotNull("Comment node cannot be null", commentContentNode);
+        final int nodeType = commentContentNode.getType();
+        Assert.assertTrue("Comment node should be a comment type",
+            TokenUtils.isCommentType(nodeType));
+        Assert.assertEquals("This should be a javadoc comment",
+            "/*", commentContentNode.getParent().getText());
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        parseTree.setParseMode(ParseMode.JAVA_WITH_JAVADOC_AND_COMMENTS);
+        final Object child = parseTree.getChild(commentContentNode, 0);
+
+        Assert.assertFalse(parseTree.isLeaf(child));
+        Assert.assertTrue(parseTree.isLeaf(tree.getFirstChild()));
+
+        final Object treeModel = parseTree.getValueAt(child, 0);
+        final String type = (String) parseTree.getValueAt(child, 1);
+        final int line = (int) parseTree.getValueAt(child, 2);
+        final int column = (int) parseTree.getValueAt(child, 3);
+        final String text = (String) parseTree.getValueAt(child, 4);
+        final String expectedText = String.join("", System.lineSeparator(),
+            "* class javadoc", System.lineSeparator(), "<EOF>");
+
+        Assert.assertEquals(null, treeModel);
+        Assert.assertEquals("JAVADOC", type);
+        Assert.assertEquals(1, line);
+        Assert.assertEquals(0, column);
+        Assert.assertEquals(expectedText, text);
+
+        try {
+            parseTree.getValueAt(child, 6);
```

Comment:
Other `getValueAt` used  `parseTree.getColumnCount()` should this be the same instead of 6?